### PR TITLE
Remove non-existent experimental effect

### PR DESCRIPTION
**Pull request by Sid127**
special_distortion_field is actually Inertial Impact
fixes https://github.com/EDCD/coriolis/issues/566
fixes https://github.com/EDCD/coriolis/issues/539

### CODE REVIEW

**Comment by Sid127** [comment]
This solves issues [#566](https://github.com/EDCD/coriolis/issues/566) and [#539](https://github.com/EDCD/coriolis/issues/566)

File: modifications/specials.json [hunk 1]
```diff
@@ -146,7 +140,7 @@
     },
     "special_inertial_impact": {
         "id": 32,
-        "edname": "special_inertial_impact",
```
Comment (felixlinker) [inline modifications/specials.json:149]: This change looks curious. So which experimental effect does not exist?

**Review by felixlinker — COMMENTED** [review]
Did you test if this breaks existing builds? E.g., could it be that there are links/saved builds out there which have the removed experimental effect on them and crash upon loading?

File: modifications/specials.json (hunk 1 above)
Comment (Sid127) [reply modifications/specials.json:149]: Distortion Field does not exist, `special_distortion_field` points to Inertial Impact

**Comment by Sid127** [comment]
I haven't tested that, I could modify things to make sure such crashes don't happen though

**Comment by Sid127** [comment]
@felixlinker fixed, tested with the build linked in https://github.com/EDCD/coriolis/issues/539

**Pull request closed without merging**